package.json

```diff
@@ -31,8 +33,15 @@
   },
   "homepage": "https://github.com/nherment/seneca-facebook-auth",
   "dependencies": {
-    "passport-facebook": "1.0.3",
-    "lodash": "~3.10.0"
+    "passport-facebook": "2.1.0",
+    "lodash": "4.2.1"
+  },
+  "devDependnecies": {
+    "eslint-config-seneca": "1.x.x",
```

Comment:
you need these three plugins for linting. so, eslint-config-standard should be changed to eslint-plugin-standard and please run the linter again, you might have more issues to fix.
eslint-config-seneca
eslint-plugin-standard
eslint-plugin-hapi